### PR TITLE
SGID Base Map & Imagery Services edits, complete

### LINKED ISSUE #805: Pages for technical writer review
- [x] [Geocoding Page](https://gis.utah.gov/developer/web-services/)
- [x] [Discover](https://gis.utah.gov/discover/)
- [x] [Aerial Photography](https://gis.utah.gov/data/aerial-photography/)
- [ ] [Base Maps](https://gis.utah.gov/data/base-map-and-imagery/)

### PR DESCRIPTION
**Pull request by atribolet**
Pull request description
------------------------

closes #805

### CODE REVIEW

File: data/base-map-and-imagery/index.md [hunk 1]
```diff
@@ -35,26 +35,26 @@ Do you need a quick, fast base map for your [GIS, CAD, or web services]({{ "/dis
...
 
 #### The Services on Discover
 
-<i class="fas fa-fw fa-map"></i> **Terrain**: This base map highlights elevation with mountain peak elevation labels, contour lines, ski resorts, as well as many other places of interest.
+<i class="fas fa-fw fa-map"></i> **Terrain**: This base map highlights elevation and features mountain peak elevation labels, contour lines, and locations of ski resort locations and other places of interest.
 
 <i class="fas fa-fw fa-map"></i> AGRC maintains a very minimal cache called **Lite**. It is very muted in color and does not distract from data overlayed on top of it.
```
Comment (atribolet) [inline data/base-map-and-imagery/index.md]: Maybe a ridiculous question, but "overlayed": so the past tense of lay is "laid," so should this be "overlaid"? Or is "overlayed" industry standard?

File: data/base-map-and-imagery/index.md [hunk 2]
```diff
@@ -35,26 +35,26 @@ Do you need a quick, fast base map for your [GIS, CAD, or web services]({{ "/dis
...
+<i class="fas fa-fw fa-map"></i> An exciting offering (_that we call squiggles and dots internally_) is a service called **Overlay**. This is a transparent set of tiles that displays styled roads and plac-enames. This can be overlayed on data for reference. Combining this with **Imagery** can create a desired **Hybrid** effect.
 
-<i class="fas fa-fw fa-map"></i> We have the **Color Infrared**, **Address Points** and **Topo** caches. The Color Infrared is a neat way to look at vegetation while the Topo cache contains all of the USGS Quad Sheets. The Topo cache displays the most relevant quad sheet (250K, 100K and 24K) depending on your scale. The Address Points cache is a fast way to view around 2.5 million address points.
+<i class="fas fa-fw fa-map"></i> We have the **Color Infrared**, **Address Points**, and **Topo** caches. The Color Infrared is a neat way to look at vegetation, while the Topo cache contains all of the USGS Quad Sheets. The Topo cache displays the most relevant quad sheet (250K, 100K and 24K) depending on your scale. The Address Points cache is a fast way to view around 2.5 million address points.
```
Comment (atribolet) [inline data/base-map-and-imagery/index.md]: Should we add an i.e., or e.g., before (250K, 100K and 24k)? Are these numbers an example or are they explanatory?

File: data/base-map-and-imagery/index.md (hunk 1 above)
Comment (steveoh) [reply data/base-map-and-imagery/index.md]: it should be overlaid

File: data/base-map-and-imagery/index.md (hunk 2 above)
Comment (steveoh) [reply data/base-map-and-imagery/index.md]: they are what is contained. I think it should be `250, 100, or 24`

**Pull request merged**